src/actors/upgrade/common/list_all_files/check

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from subprocess import Popen, PIPE
+import json
+import sys
+
+
+def log_error(errmsg):
+    """Log error on stderr."""
+    sys.stderr.write("%s\n" % errmsg)
+
+
+# NOTE: expected dirs without whitespaces
+cmd_local_mnt = "df --local -P | rev | cut -f1 -d' ' | rev | tail -n +2"
+cmd_file_list = "find $i -xdev -print"
+cmd = "for i in $(%s);do %s; done |sort |uniq" % (cmd_local_mnt, cmd_file_list)
+
+p = Popen(cmd, shell=True, stdout=PIPE)
+p_out, dummy = p.communicate()
```

Comment:
instead of having ```dummy``` why not to use ```_``` ?

Reply:
Because `_` is overloaded in python. It is one of allowed ways how to mark variable as unused, but it is even name of function in some python library (`gettext`?). I've got used to it over time as it's pretty clean what it is, doesn't matter on current namespace.

Reply:
yes it's ```gettext```, i prefer use underscore although it's your way so ok.

Reply:
I still prefer dummy name in Python, but in case we will want to unify code style I can use underline instead as well in actors.